### PR TITLE
[YUNIKORN-1923] update apps columns

### PR DESCRIPTION
**Pull request by FrankYang0529**
### What is this PR for?

* remove partition and queue columns
* add pending resource column

### What type of PR is it?
* [X] - Improvement

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-1923

### Screenshots (if appropriate)

Screenshot 2023-08-22 at 4 25 31 PM

### CODE REVIEW

**Review by craigcondit — CHANGES_REQUESTED** [review]
Can we get some test data that exercises this in the json db ?

**Comment by FrankYang0529** [comment]
Hi @craigcondit, I added a test case for it. However, I'm not familiar with Angular. Not sure whether I did it right. Could you help me review again? Thank you.

**Comment by craigcondit** [comment]
@FrankYang0529 not just the test case, but the example JSON data that is used to drive json-db. This allows us to visually see the impact of the changes in the UI.

**Comment by craigcondit** [comment]
@FrankYang0529 can you fix the merge conflicts and update json-db with non-zero pending resources?

**Comment by FrankYang0529** [comment]
> @FrankYang0529 can you fix the merge conflicts and update json-db with non-zero pending resources?

Hi @craigcondit, thanks for the review. Also, sorry for late. I will resolve conflicts and update json-db tomorrow 👍 .

**Comment by FrankYang0529** [comment]
> > @FrankYang0529 can you fix the merge conflicts and update json-db with non-zero pending resources?
> 
> Hi @craigcondit, thanks for the review. Also, sorry for late. I will resolve conflicts and update json-db tomorrow 👍 .

Hi @craigcondit, I resolved conflicts and add non-zero pending resources to json-db. Could you review it again when you have time? Thank you.

File: json-db.json [hunk 1]
```diff
@@ -512,6 +512,46 @@
       "hasReserved": false,
       "reservations": [],
       "maxRequestPriority": -2147483648
+    },
+    {
+      "applicationID": "application-sleep-0002",
```
Comment (0yukali0) [inline json-db.json:517]: There is already an app-0002 in the database.
For distinguishing the applications, replace 0002 with 0006.

**Review by 0yukali0 — CHANGES_REQUESTED** [review]
I ran the test and the `make start-dev` locally to check the result.
The result meets the description in this pull request.

File: json-db.json (hunk 1 above)
Comment (FrankYang0529) [reply json-db.json]: Updated it. Thank you.

**Comment by wilfred-s** [comment]
This looks like it is all done. Can we get a final review of the changes as I think all requested changes are made

**Review by wilfred-s — APPROVED** [review]
LGTM, +1

**Pull request closed without merging**